CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Added compatibility with future crate versions.
```

Comment:
... with future crate versions which will serve the admin ui from /admin/ instead of `_plugins...